### PR TITLE
Improve application usability

### PR DESCRIPTION
**Pull request by Oleh-Stefanovych**
Major changes:
- Now, when click opens a details block. A double click opens a folder.
- Native component for selecting an avatar file on the registration page has been replaced with a custom one.
- Minor changes of paddings and border radius.

### CODE REVIEW

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

File: app/component/droppable-component/file-item/index.js [hunk 1]
```diff
@@ -51,11 +51,20 @@ export class FileItem extends DroppableComponent {
    */
   _addEventHandlers() {
     if (this._file.type === 'folder') {
+      let clicks = 0;
       this.element.addEventListener('click', () => {
-        const isFolderCreation = this._file.id === undefined;
-        if (!isFolderCreation && !this._isRenaming) {
```
Comment (dkalinin) [inline app/component/droppable-component/file-item/index.js]: Click on item in creation folder mode will open details for the folder which is not created yet.

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, fixed, PTAL.

**Fixes:**
- Fix bug with unexpected opening details for uncreated folder
- Fix bug with unexpected opening details for renaming file
- Fix bug with a delay when opening details for a folder

**Note:**
- To start the server, please use `npm run ds`. Application will be available on [http://localhost:5001/](http://localhost:5001/).
- There is a situation when you try to open a folder in which you are already. A redirect to the same address occurs. Visible opening of the folder does not occur. This happens because the same files are always returned from the server.

The `ESLint` result:
image

File: app/component/droppable-component/file-item/index.js [hunk 2]
```diff
@@ -51,28 +51,26 @@ export class FileItem extends DroppableComponent {
...
       const isFolderCreation = this._file.id === undefined;
       if (isFolderCreation) {
         this.element.addEventListener('dragenter', (event) => this._stopAndPrevent(event));
         this.element.addEventListener('dragleave', (event) => this._stopAndPrevent(event));
         this.element.addEventListener('drop', (event) => this._stopAndPrevent(event));
       } else {
+        let clicks = 0;
+        this.element.addEventListener('click', () => {
+          if (clicks === 0) {
+            this._onShowFileDetailsCallback && this._onShowFileDetailsCallback(this._file);
```
Comment (dkalinin) [inline app/component/droppable-component/file-item/index.js]: It opens details each time when user tries to navigate to this folder.

Comment (Oleh-Stefanovych) [reply app/component/droppable-component/file-item/index.js]: I find it more attractive than the delay in opening details.

**Pull request merged**